cv32e20/bsp/handlers.S

```diff
@@ -68,7 +69,7 @@
 .weak m_fast13_irq_handler
 .weak m_fast14_irq_handler
 .weak m_fast15_irq_handler
-
+.weak m_nmi_irq_handler
```

Comment:
The `debug_test` wants to overload `__no_irq_handler`, so please add the following line below this one:
```
.weak __no_irq_handler
```

Reply:
I put this line in at a different spot (line 52).  It works for me. I must not have simulated the debug_test prior to the last change.  It works for me now.